media/js/base/sentry-consent.es6.js

```diff
@@ -12,7 +12,7 @@ import {
     getCurrentScope,
     globalHandlersIntegration,
     httpContextIntegration,
-    inboundFiltersIntegration,
+    eventFiltersIntegration,
```

Comment:
`inboundFiltersIntegration` was renamed and will be removed in future version, we're making the update now
<img width="755" height="255" alt="deprecated-name" src="https://github.com/user-attachments/assets/9c2cf2e5-ab65-44b6-bcc9-93531f351828" />